src/scenes/Species/SpeciesProjectsTable.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+
+import { Grid, Typography } from '@mui/material';
+import { TableRowType } from '@terraware/web-components';
+import { TableColumnType } from '@terraware/web-components/components/table/types';
+
+import Table from 'src/components/common/table';
+import { useOrganization } from 'src/providers';
+import { SearchService } from 'src/services';
+import strings from 'src/strings';
+import { SearchRequestPayload } from 'src/types/Search';
+
+import RemoveProjectsDialog from './RemoveProjectsDialog';
+
+const columns = (): TableColumnType[] => [
+  { key: 'projectName', name: strings.PROJECT, type: 'string' },
+  { key: 'submissionStatus', name: strings.STATUS, type: 'string' },
+];
+
+type SpeciesProjectsTableProps = {
+  speciesId: number;
+};
+
+type SpeciesProjectsResult = {
+  id: number;
+  submissionStatus: string;
+  projectName: string;
+};
+
+type SpeciesProjectsSearchResult = {
+  participantProjectSpecies: { id: number; submissionStatus: string; project: { name: string } }[];
+};
+
+export default function SpeciesProjectsTable({ speciesId }: SpeciesProjectsTableProps): JSX.Element {
+  const { selectedOrganization } = useOrganization();
+  const [searchResults, setSearchResults] = useState<SpeciesProjectsResult[] | null>();
+  const [selectedRows, setSelectedRows] = useState<TableRowType[]>([]);
+  const [showRemoveDialog, setShowRemoveDialog] = useState(false);
+  const [reload, setReload] = useState(false);
+
+  useEffect(() => {
+    const getApiSearchResults = async () => {
```

Comment:
Can you plumb this all through redux? Maybe add a function to the `ParticipantProjectSpeciesService` like `getProjectsForSpecies` or similar.

Reply:
You are right!